owl/EASE-OBJ.owl

```diff
@@ -69,6 +280,10 @@
         <owl:inverseOf rdf:resource="http://www.ease-crc.org/ont/EASE-OBJ.owl#isLocalizationOf"/>
         <rdfs:domain rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#PhysicalObject"/>
         <rdfs:range rdf:resource="http://www.ease-crc.org/ont/EASE-OBJ.owl#Localization"/>
+        <owl:propertyChainAxiom rdf:parseType="Collection">
```

Comment:
Just as in the case of hasColor, this appears similarly inconsistent with the DUL foundation. If a Localization is a Quality, then it cannot be in the range of isClassifiedBy, which is Concept.